### PR TITLE
escape angle brackets in response header

### PR DESCRIPTION
**Pull request by steffkes**
Not sure about what happend to the files in `dist`, that looks a bit weird for such a small change, but i follow the steps in the readme, did `npm install` and `npm run-script build` and that is what came out :|

I realized the problem, after adding an `Link`-Header like described in [RFC5988](http://tools.ietf.org/html/rfc5988#section-5.5), which then looks like:

    Date: Mon, 17 Jun 2013 17:59:54 GMT
    Content-Encoding: gzip
    Transfer-Encoding: chunked
    Content-Type: application/json
    Link: <https://api.project.tld/transactions?page=2>; rel="next"

Since the `showStatus` method in `OperationView.coffee` just puts the response headers between `<pre>` Tags, the Link is actually treated as an unknown html-tag and not displayed in the browser.

i saw there were several escape functions in the code .. but some where hidden inside the handlebar code, others did to much in case of escaping so i used two simple `replace` calls instead.

so, having said that .. let me know what you think about it :)

### CODE REVIEW

File: src/main/coffeescript/view/OperationView.coffee [hunk 1]
```diff
@@ -261,7 +261,7 @@ class OperationView extends Backbone.View
     response_body = pre
     $(".response_code", $(@el)).html "<pre>" + data.status + "</pre>"
     $(".response_body", $(@el)).html response_body
-    $(".response_headers", $(@el)).html "<pre>" + data.getAllResponseHeaders() + "</pre>"
+    $(".response_headers", $(@el)).html "<pre>" + data.getAllResponseHeaders().replace( /</g, '&lt;' ).replace( />/g, '&gt;' ) + "</pre>"
```
Comment (mohsen1) [inline src/main/coffeescript/view/OperationView.coffee]: escaping HTML is a little more than just replacing `<` and `>`. Please use `$('<div/>').text(data.getAllResponseHeaders()).html()` to get escaped HTML

**Comment by mohsen1** [comment]
Please rebase your changes to `develop_2.0` branch and fix the issue that I commented in your code then make another PR to `develop_2.0` branch.

Thanks you!

**Comment by steffkes** [comment]
i'm currently not having the time to update this PR, if anyone else does - please feel free to pick it up. @matt-thomson maybe?

**Comment by mohsen1** [comment]
I can make this change and close this PR. Are you okay with that?

**Comment by steffkes** [comment]
sure, go ahead

**Comment by mohsen1** [comment]
Actually it's already using `_.escape` now

https://github.com/swagger-api/swagger-ui/blob/develop_2.0/src/main/coffeescript/view/OperationView.coffee#L447

**Pull request closed without merging**